webapp/src/utils/url_utils.js

```diff
@@ -1,9 +1,7 @@
 export const isValidUrl = (urlString) => {
-    let url;
     try {
-        url = new URL(urlString);
+        return new URL(urlString);
```

Comment:
This function has an inconsistent return type. We can do this:

```suggestion
        return Boolean(new URL(urlString));
```